src/content/blog/2024-05-17-utah-housing-affordability-map.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: 'Mapping Utah’s Housing Affordability'
+author: Greg Bunce
+date: 2024-05-17T15:00:00.000Z
+category: Mapping
```

Comment:
**issue**: this is not a valid blog category

Reply:
do we have an official list of categories?

Reply:
based on the other posts, UGRC seems fitting, aye?